client/package.json

```
@@ -21,6 +23,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.2",
     "react-scripts": "5.0.1",
+    "react-stripe-checkout": "^2.6.3",
```

```suggestion
    "react-stripe-checkout": "^2.6.3",
```
uninstall this package as we don't use it